### PR TITLE
Testing farm

### PR DESCRIPTION
**Pull request by obabec**

### CODE REVIEW

File: .packit.yaml [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+
+  ###############################################################################################
+  - job: tests
+    trigger: pull_request
+    # Suffix for job name
+    identifier: "postgres-12-pgoutput"
+    targets:
+      # This target is not used at all by our tests, but it has to be one of the available - https://packit.dev/docs/configuration/#aliases
+      - centos-stream-8-x86_64
+    skip_build: true
```
Comment (lachmanfrantisek) [inline .packit.yaml]: Hello, Packit PO here..;)

If I haven't missed any (re)definition on TMT/FMF level, this is actually used to specify the environment (VM) that your tests run on.

Comment (obabec) [reply .packit.yaml]: yes, that's true. Thank you for pointing that out :)

File: .packit.yaml [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+    identifier: "postgres-12-decoderbufs"
+    targets:
+      - centos-stream-8-x86_64
+    skip_build: true
+    manual_trigger: true
+    labels:
+      - postgres
+    tf_extra_params:
+      test:
+        fmf:
```
Comment (mfocko) [inline .packit.yaml]:
```suggestion
        tmt:
```

Hi, sorry for the trouble, Testing Farm is deprecating the `test.fmf` in favor of `test.tmt` and we have already switched to it. You should get it running by switching here and below to `tmt`.

Comment (obabec) [reply .packit.yaml]: Thanks, done. Seems to be ok now

**Pull request closed without merging**